tools/lint-md.js

```diff
@@ -49762,8 +49762,8 @@ function prohibitedStrings (ast, file, strings) {
         results.forEach(({ result, index }) => {
           const message = val.yes ? `Use "${val.yes}" instead of "${result}"` : `Do not use "${result}"`;
           file.message(message, {
-            start: location.toPosition(initial + index),
-            end: location.toPosition(initial + index + [...result].length)
+            start: location.toPoint(initial + index),
+            end: location.toPoint(initial + index + [...result].length)
```

Comment:
Is this generated or did you read the docs/rel notes and change it manually? Just curious.

Reply:
This file is generated from `tools/node-lint-md-cli-rollup`